.github/workflows/build_develop.yml

```diff
@@ -102,7 +102,7 @@ jobs:
 
             - name: Deploy to Cloudflare Pages
               id: cfp
-              uses: cloudflare/pages-action@1
+              uses: cloudflare/pages-action@v1
```

Comment:
We should probably lock this down to a sha like we do for most other 3rd party actions